public/app/features/annotations/components/StandardAnnotationQueryEditor.tsx

```diff
@@ -177,8 +188,10 @@ export default class StandardAnnotationQueryEditor extends PureComponent<Props,
           onRunQuery={this.onRunQuery}
           data={response?.panelData}
           range={getTimeSrv().timeRange()}
+          annotation={annotation}
+          onAnnotationChange={this.onAnnotationChange}
         />
-        {datasource.type !== 'datasource' && (
+        {!datasource.annotations!.dontUseMapping && (
```

Comment:
Maybe:
```suggestion
        {!datasource.annotations?.useLegacyRunner && (
```
If we add `dontUseMapping` i would expect it to also affect if the mappins processed (not just shown)

Reply:
Mentioned here https://github.com/grafana/grafana/pull/48814/files#r879183983.

Regarding the processing I am not sure what do you mean.